packages/featureflags/featureToggleManager.js

```diff
@@ -40,6 +40,8 @@ class FeatureToggleManager {
                 redisClient.set(REDIS_FEATURES_KEY, JSON.stringify(fetchedFeatureFlags), 'EX', FEATURES_EXPIRED_IN_SECONDS)
 
                 return fetchedFeatureFlags
+            } else {
+                return {}
```

Comment:
Now the empty object means no configuration data. It looks strange :)
What about throwing an error in the case of no configuration data?
```
if (!featureToggleApiUrl || !featureToggleApiKey) {
    throw new Error('no config message')
}
```

Also, it may be helpful to return an empty object from the catch-branch.

Reply:
done